classes/bootspec.bbclass

```diff
@@ -36,15 +36,17 @@ BOOTSPEC_EXTRALINE ?= ""
 BOOTSPEC_EXTRALINE[doc] = "Allows to add extra content to bootspec entries, lines must be terminated with a newline"
 
 python create_bootspec() {
-    dtb = (d.getVar('KERNEL_DEVICETREE') or "default").replace('.dtb', '').split()
```

Comment:
Shouldn't we split this commit in two?

Reply:
No strong opinion on that. A separate one for the 'dtbs'-Rename? Or for the conf variable?

Reply:
That would be cool :)

Reply:
I like people answering with 'yes' on an 'or' question! ;)

Reply:
Damn. I mean: one patch for the conf in front of the renaming.

Reply:
Ok, did that.